### PR TITLE
create documentation for hl7-fhir transforms

### PR DESCRIPTION
**Pull request by GilmoreA6**
This PR adds documentation for what types of fhir->hl7 transforms are being used by engagement.

### CODE REVIEW

File: prime-router/docs/onboarding-users/fhir-to-hl7-transforms.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+## Types of FHIR->HL7 transforms currently used
+
+Default value in field - Default a static value into a single field or subfield
+```yaml
+  - name: ak-receiving-application
+    value: [ '"AKDOH"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+```
+
+Removing single field - Remove any value in a single field or subfield
```
[the remark this answers is not in the document]
Comment (GilmoreA6) [reply prime-router/docs/onboarding-users/fhir-to-hl7-transforms.md:12]: it's technically setting the value to empty, but an HL7 V2 parser treats null vs empty as the same thing. I will change the description in the interests of being precise

[the remark this answers is not in the document]
Comment (GilmoreA6) [reply prime-router/docs/onboarding-users/fhir-to-hl7-transforms.md:12]: Strictly speaking you can send an explicit NULL value by sending "" in the field (which we don't do here), but that only matters in certain contexts and depends on how the receiver has configured their application to update existing records

[the remark this answers is not in the document]
Comment (GilmoreA6) [reply prime-router/docs/onboarding-users/fhir-to-hl7-transforms.md:12]: Null and empty fields: The null value is transmitted as two double quote marks (“”). A null-valued field
differs from an empty field. An empty field should not overwrite previously entered data in the field, while
the null value means that any previous value in this field should be overwritten.
Value in Field Meaning
“”
|””|
Nullify the value recorded in the receiving system data base.
<empty field>
||
Make no changes to the record in the receiving data base. The sending sy

**Review by oslynn — APPROVED** [review]

**Review by JessicaWNava — APPROVED** [review]
LGTM

**Review by lucero-v — APPROVED** [review]

**Review by victor-chaparro — APPROVED** [review]

**Pull request merged**